doc/source/old_sections/ad_inst_ta1.rst

```diff
@@ -298,7 +298,7 @@ with the HDFS name node.
     $ sudo yum -y install trustedanalytics-rest-server
 
 Installing On A Worker Node
----------------------------
+-
 
 The |PACKAGE| spark dependencies package needs to be installed on every node
```

Comment:
This section  needs update.
There are no more spark dependencies.
It was removed a while ago.